pubspec.yaml

```diff
@@ -1,35 +1,38 @@
 name: flutter_lokalise
-version: 0.1.4
+version: 1.0.0
 description: >
   A tool to download translation files from Lokalise and convert them to .arb files.
 homepage: https://github.com/meniga/flutter_lokalise
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 executables:
   flutter_lokalise:
 
 dependencies:
-  archive: ">=2.0.0 <3.0.0"
-  args: ">=1.5.0 <2.0.0"
-  json_annotation: ">=3.0.0 <4.0.0"
-  freezed_annotation: ">=0.7.0 <2.0.0"
-  http: ">=0.12.0 <1.0.0"
-  logging: ">=0.11.0 <1.0.0"
-  meta: ">=1.1.0 <2.0.0"
-  path: ">=1.5.0 <2.0.0"
-  quiver: ">=2.0.0 <3.0.0"
-  yaml: ">=2.0.0 <3.0.0"
+  archive: ^3.1.2
+  args: ^2.1.0
+  json_annotation: ^4.0.1
+  freezed_annotation: ^0.14.1
+  http: ^0.13.3
+  logging: ^1.0.1
+  meta: ^1.3.0
+  path: ^1.8.0
+  quiver: ^3.0.1
+  yaml: ^3.1.0
 
 dev_dependencies:
-  build_runner: ^1.10.1
-  json_serializable: ^3.4.0
-  freezed: ^0.12.1
-  matcher: ^0.12.9
-  mock_web_server: ^4.1.1
-  mockito: ^4.1.1
-  pedantic: ^1.9.2
-  remove_from_coverage: ^1.0.7
-  test: ^1.15.2
-  test_coverage: ^0.4.3
+  build_runner: ^2.0.2
+  json_serializable: ^4.1.1
+  freezed: ^0.14.1
+  matcher: ^0.12.10
+  mock_web_server: ^5.0.0-nullsafety.1
+  mockito: ^5.0.7
+  pedantic: ^1.11.0
+  remove_from_coverage: ^2.0.0
+  test: ^1.17.3
+  test_coverage:
+    git:
```

Comment:
Git dependencies can't be used in packages published in pub.dev.

Reply:
Any suggestions?

Reply:
We can use `flutter test --coverage` instead of `flutter pub run test_coverage --no-badge` in `.github/workflows/ci.yaml`. Then `test_coverage` package will not be needed.